agr_api/src/main/java/org/alliancegenome/api/controller/AlleleController.java

```diff
@@ -133,32 +141,32 @@ public JsonResultResponse<PhenotypeAnnotation> getPhenotypePerAllele(String id,
 		}
 	}
 
-   @Override
+	@Override
 	public Response getPhenotypesPerAlleleDownload(String id,
 												   String phenotype,
 												   String source,
 												   String reference,
 												   String sortBy) {
-		JsonResultResponse<PhenotypeAnnotation> response = getPhenotypePerAllele( id,
-				Integer.MAX_VALUE,
-				1,
-		 phenotype,
-		 source,
-		 reference,
-		 sortBy);
+		JsonResultResponse<PhenotypeAnnotation> response = getPhenotypePerAllele(id,
+			Integer.MAX_VALUE,
+			1,
+			phenotype,
+			source,
+			reference,
+			sortBy);
 		Response.ResponseBuilder responseBuilder = Response.ok(phenotypeAnnotationToTdfTranslator.getAllRowsForAlleles(response.getResults()));
 		APIServiceHelper.setDownloadHeader(id, EntityType.ALLELE, EntityType.PHENOTYPE, responseBuilder);
 		return responseBuilder.build();
 	}
 
-	public JsonResultResponse<DiseaseAnnotation> getDiseasePerAllele(String id,
-																	 Integer limit,
-																	 Integer page,
-																	 String disease,
-																	 String source,
-																	 String reference,
-																	 String associationType,
-																	 String sortBy) {
+	public JsonResultResponse<DiseaseAnnotation> getDiseasePerAlleleOld(String id,
```

Comment:
If this doesn't get used anymore... it needs to be deleted. if you really need to look at this code in the future it can be pulled from history.

Reply:
Yes.